internal/transform/transform.go

```diff
@@ -262,15 +262,25 @@ func isRawElement(n *astro.Node) bool {
 	if n.Type == astro.FrontmatterNode {
 		return true
 	}
+	for _, attr := range n.Attr {
+		if attr.Key == "is:raw" {
+			return true
+		}
+	}
```

Comment:
It looks like we can extract this out of the `rawTags` loop as it checks against the `n` node only.